workalendar/america/brazil.py

```diff
@@ -304,14 +302,14 @@ class BrazilSantaCatarina(Brazil):
 class BrazilSaoPauloState(Brazil):
     "Brazil São Paulo State"
     FIXED_HOLIDAYS = Brazil.FIXED_HOLIDAYS + (
-        (7, 9, "Constitutional Revolution of 1932"),
+        (7, 9, "Revolução Constitucional de 1932"),
     )
 
 
 class BrazilSaoPauloCity(BrazilSaoPauloState):
     "Brazil São Paulo City"
     FIXED_HOLIDAYS = BrazilSaoPauloState.FIXED_HOLIDAYS + (
-        (1, 25, "Anniversary of the city of São Paulo"),
+        (1, 25, "Aniversário da cidade de São Paulo"),
```

Comment:
It's missing 11/20 - "Dia de Zumpi e Consciência Negra"

Reply:
"Dia da Consciência Negra" is already in this calendar, it comes from the calendar of the state of São Paulo.